renderdoc/driver/vulkan/wrappers/vk_misc_funcs.cpp

```diff
@@ -1583,6 +1583,24 @@ void WrappedVulkan::vkDebugReportMessageEXT(VkInstance instance, VkDebugReportFl
                                                   location, messageCode, pLayerPrefix, pMessage);
 }
 
+void WrappedVulkan::vkSetHdrMetadataEXT(VkDevice device, uint32_t swapchainCount,
+                                        const VkSwapchainKHR *pSwapchains,
+                                        const VkHdrMetadataEXT *pMetadata)
+{
+  vector<VkSwapchainKHR> unwrappedSwaps;
+  for(uint32_t i = 0; i < swapchainCount; i++)
+    unwrappedSwaps.push_back(Unwrap(pSwapchains[i]));
```

Comment:
FYI there is `UnwrapArray` which is useful for this type of pattern, it uses TLS memory to avoid local allocs. Not really important for a function off the hot path like this, though.